index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Apple</title>
+</head>
+
+<body>
+    <main id="core">
+        <div id="core-section">
+            <nav id="top-bar">
+                <a class="nav-links" href="#"><img id="nav-logo" src="assets/apple.svg" alt="Apple Logo">
+                </a>
+                <a class="nav-links" href="#">
+                    Store</a>
+                <a class="nav-links" href="#">
+                    Mac</a>
+                <a class="nav-links" href="#">
+                    iPod</a>
+                <a class="nav-links" href="#">
+                    iPhone</a>
+                <a class="nav-links" href="#">
+                    iPad</a>
+                <a class="nav-links" href="#">
+                    iTunes</a>
+                <a class="nav-links" href="#">
+                    Support</a>
+                <div class="nav-search">
```

Comment:
**[OPTIONAL]** Add a [search icon](https://fontawesome.com/icons/search?style=solid) to the [search field.](https://www.w3schools.com/howto/howto_css_searchbar.asp).